.envrc

```diff
@@ -120,7 +120,7 @@ export SENTRY_UI_HOT_RELOAD=1
 if [ "$SENTRY_DEVENV_NO_REPORT" ]; then
     info "No development environment errors will be reported (since you've defined SENTRY_DEVENV_NO_REPORT)."
 else
-    info "Development errors will be reported to Sentry.io. If you wish to opt-out, set SENTRY_DEVENV_NO_REPORT as an env variable."
+    info "Development errors will be reported to Sentry.io."$'\n'"        If you wish to opt-out, set SENTRY_DEVENV_NO_REPORT as an env variable."
```

Comment:
maybe I forgot and there's some better more obvious way to insert a newline, but you def can't just slam a `\n` in there because of the heredoc